src/plugins/data/public/plugin.ts

```diff
@@ -117,19 +116,21 @@ export class DataPublicPlugin
       storage: this.storage,
     });
 
+    uiActions.registerTrigger(applyFilterTrigger);
+
     uiActions.registerAction(
       createFilterAction(queryService.filterManager, queryService.timefilter.timefilter)
     );
 
     uiActions.addTriggerAction(
-      SELECT_RANGE_TRIGGER,
+      'SELECT_RANGE_TRIGGER',
```

Comment:
There wasn't a great way to keep these trigger IDs as consts since they lived in `embeddable`, so I just used the ID string here.